Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10-slim
```

Comment:
Changing this to specify bookworm because the Microsoft ODBC driver install has to target a specific version of debian.